assets/src/onboarding-wizard/pages/done/use-preview.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { useContext, useMemo, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import { PREVIEW_URLS } from 'amp-settings'; // From WP inline script.
+
+/**
+ * Internal dependencies
+ */
+import { Options } from '../../../components/options-context-provider';
+import { STANDARD } from '../../../common/constants';
+
+/**
+ * Gets the title for the preview page selector.
+ *
+ * @param {string} type The page type.
+ */
+function getTitle( type ) {
+	switch ( type ) {
+		case 'home':
+			return __( 'Homepage', 'amp' );
+
+		case 'author':
+			return __( 'Author page', 'amp' );
+
+		case 'date':
+			return __( 'Archive page', 'amp' );
+
+		case 'search':
+			return __( 'Search results', 'amp' );
+
+		default:
+			return `${ type.charAt( 0 ).toUpperCase() }${ type.slice( 1 ) }`;
```

Comment:
This has been addressed in eca7274.

Note that we may still have issues with the custom taxonomy and the custom post type labels. For instance, WooCommerce registers a `product_cat` taxonomy. Its `name` is "Product categories" and its `singular_name` is just "Category". The core Category `name` is "Categories" and the `singular_name` is "Category", too.

I went with the `name` label since it best differentiates the taxonomies. The drawback is that it is usually a plural noun and probably should be used standalone (unless we're okay with having "Categories Archive" or "Tags Archive").